Makefile.PL

```diff
@@ -351,7 +351,7 @@ if (-d 'inc/patches') {
     if (which('patch')) {
         for my $file (glob 'inc/patches/*.patch') {
             # replacing \n with \n will change \n to \r\n on Win, without this patch.exe will crash
-            system "cd \Q$mxbase\E && perl -pe \"s/\\n/\\n/\" ../../\Q$file\E | patch -p1 -N";
+            system "cd $mxbase && perl -pe \"s/\\n/\\n/\" ../../$file | patch -p1 -N";
```

Comment:
`\Q…\E` here was used for security - to avoid occasional injects into command line.
Why do you think it works incorrectly on Windows (I've tested it and everything was fine)?
To make this code safe without `\Q…\E` you need to check `$mxbase` and `$file` first to make sure they didn't contain any special chars (actual check should allow only safe chars, not try to reject special chars). For ex.:
```perl
die if $mxbase !~ m{\A[A-Za-z0-9_./-]+\z}ms;
die if $file   !~ m{\A[A-Za-z0-9_./-]+\z}ms;
```